### ISSUE TITLE
(This port is not an HTTP proxy; did you want to use HTTPTunnelPort?)

### ISSUE THREAD

**Issue by Yquas** [Gedsh/InviZible#84]
The in-app Tor log on my phone is always full of the warnings you see in the title of this issue, with some varying text before it. Since I began using InviZible Pro it was always like that, the log is pretty much consisting just of that. Every single day.
How can this be fixed? Maybe just some settings?
Here's a picture of it:
![Screenshot_2021-04-05-09-12-24-399_pan alexander tordnscrypt](https://user-images.githubusercontent.com/81969859/113698865-2ea99c00-96d5-11eb-9ee2-083a61d367b2.png)

**Comment by Gedsh** [comment]
It looks like you have configured some application to use the Tor socks proxy. But this application requires an HTTP proxy. So find this app and change the port from 9050 to 8118.

**Comment by Yquas** [comment]
> It looks like you have configured some application to use the Tor socks proxy. But this application requires an HTTP proxy. So find this app and change the port from 9050 to 8118.

Oh, ok. Many of my apps are using the socks proxy, I thought they all support it when they don’t complain. But I’ll take a look what it could be, thanks.

**Comment by Yquas** [comment]
I really have no idea what app could be causing this. There aren’t many running all the time (I guess that has to be fulfilled for it to spam my log all day and night) and out of those, none fit. Either they’re set to socks or they have a builtin proxy I cannot control. Some thoughts are FairEmail (didn’t find a setting for a proxy) and (a fork of) Conversations (set to route through Tor but with no additional settings possible).

**Comment by Gedsh** [comment]
> I really have no idea what app could be causing this.

But I really can't help you find it :)

> Or is it just maybe possible InviZible Pro is causing it by itself?

Nope. It is definitely a configuration problem for some of your applications.

**Comment by Yquas** [comment]
Ok, I’ll be looking at it in my free time this week.

**Comment by Yquas** [comment]
I could not find anything even after two weeks. And changing the port would break some things. But I'll probably try it.

**Comment by Gedsh** [comment]
As I understand it, you will wait while I do it for you? :D

**Comment by Yquas** [comment]
Nope. I just want ro figure out this problem some day properly and not go around it.

**Comment by Gedsh** [comment]
> changing the port would break some things.

Why don't you want to change the Socks port? What will be broken?

> but if you or someone else has an idea, I'd be very happy to hear it

I gave you an idea. You don't even need to search for a spamming app.

**Comment by marcdw1289** [comment]
Searching the web for the error in the screenshot, "socks version 67 not recognized" turns up some interesting info.
As the dev points out there must be a misconfigured app or browser causing those entries.

**Comment by Yquas** [comment]
@marcdw1289 Thanks, but that's also what Gedsh already told me, pretty much. My problem os that I don't know the app causing it and desperately want to find it.. But I had no success so far, and I've done everything I can. I'm still on the way of lookimg at Gedshs other tip, changing the SOCKS port. But I have no time for these things right now.

**Comment by Yquas** [comment]
Sorry for not testing it further in time.
But I think I found out the problem. I had the Proxy setting in Android Wi-Fi settings set to localhost and 9050 on my home network turned on since long ago, thinking it would be more secure or whatever.
After removing that yesterday, setting the proxy to none, the HTTP protocol spam completely vanished.

But there's a problem with InviZible right now: I found out I should turn this off because InviZible itself told me that Android feature would interfere with it. But this warning never showed up before, it only appeared when InviZible was on AND I changed the setting to none (because I needed to test something else) when it was running. But in other cases, starting InviZible never showed this warning.
Is it possible for InviZible to check the Android proxy setting every time and not only when the setting is changed when InviZible is running? That would be a more effective way of warning.
